data-structures-stacks/count-values.js

```diff
@@ -0,0 +1,13 @@
+/* exported countValues */
+
+function countValues(stack) {
+  if (stack.print().split('<-').length === 1) {
+    return 0;
+  } else {
+    return stack.print().split('<-').length;
+  }
```

Comment:
Try to write your solutions without printing the stack. that method is just for debugging. check your other solutions too.